CodeEdit/Commands/CommandPaletteState.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  CommandPaletteState.swift
+//  CodeEdit
+//
+//  Created by Alex on 25.05.2022.
+//
+
+import Foundation
+import Keybindings
+
+public final class CommandPaletteState: ObservableObject {
```

Comment:
Please add proper documentation to `public` classes, structs, methods and variables/constants.

A guide about code documentation can be found [here](https://nshipster.com/swift-documentation/).